examples/v2/parse.py

```diff
@@ -1,16 +1,16 @@
-import config
+from .. import config
 
 config.setup_examples()
 import infermedica_api
 
 
 if __name__ == '__main__':
-    api = infermedica_api.get_api()
+    api = infermedica_api.get_api('v2')
 
     print('Parse simple text:')
-    response = api.parse('i feel smoach pain but no couoghing today')
+    response = api.parse('i feel smooch pain but no coughing today')
```

Comment:
Q: have these examples stopped working? :D

Reply:
No but at this point I don't remember if the typos were intentional or not so decided to fix them.